README.md

```diff
@@ -56,6 +62,7 @@ with:
   TOUCHPOINT_TAGS: "e53621bc-f66a-49f0-a886-537b5c64df22"
   TOUCHPOINT_TYPE: "asdff3e5-cd3d-af42-ax3c-adsf2342c324"
   TOTANGO_USER_NAME: "totango_user_email@email.com"
```

Comment:
Do you mind changing this one to TASK_ASSIGNEE while we're here? I think I might have put this on the wrong line :D It's supposed to be on 64